pages/api/turso-global.ts

```diff
@@ -0,0 +1,55 @@
+import { NextRequest as Request, NextResponse as Response } from "next/server";
+import { Client as LibsqlClient, createClient } from "@libsql/client/web";
+
+
+export const config = {
+  runtime: "edge",
+};
+
+const start = Date.now();
+const client = buildLibsqlClient();
+
+function buildLibsqlClient(): LibsqlClient {
+    const url = process.env.TURSO_DB_URL?.trim();
+    if (url === undefined) {
+        throw new Error("TURSO_DB_URL env var is not defined");
```

Comment:
You can DM me the credentials on Twitter or lee at vercel dot com.

Reply:
Done. Please let me know if you got them.